File: Formations/Boucles/Random Loops/Solutions/3/3.java
import java.util.Random;
import java.util.Scanner;

class Solution {
    public static void main(String[] args) {
        int score = 0;
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        
        for (int i = 0; i < 10; i++) {
            int a = random.nextInt(10) + 1;
            int b = random.nextInt(10) + 1;
            int rep = a * b;
            
            System.out.print(a + " x " + b + " = ?");
            int sugg = scanner.nextInt();
            
            if (rep == sugg) {
                score++;
            } else {
                score--;
            }
        }
        System.out.println("Your Score = " + score);
        scanner.close();
    }
}

File: Formations/Boucles/Random Loops/Solutions/3/3.js
let score = 0;

for (let i = 0; i < 10; i++) {
    let a = Math.floor(Math.random() * 10) + 1;
    let b = Math.floor(Math.random() * 10) + 1;
    let rep = a * b;
    let sugg = parseInt(prompt(`${a} x ${b} = ?`));
    if (rep === sugg) {
        score++;
    } else {
        score--;
    }
}
console.log(`Your Score = ${score}`);
